sjsonnet/src/sjsonnet/YamlRenderer.scala

```diff
@@ -58,7 +58,7 @@ class YamlRenderer(_out: StringWriter = new java.io.StringWriter(), indentArrayI
       }
       depth -= 1
     } else {
-      upickle.core.RenderUtils.escapeChar(null, elemBuilder, s, unicode=true)
+      upickle.core.RenderUtils.escapeChar(null, elemBuilder, s, escapeUnicode=true, wrapQuotes = true)
```

Comment:
note: pay attention here `wrapQuotes = true` comes from lib update. I checked and I think true is a valid value